docs/control-structures.rst

```diff
@@ -28,6 +28,9 @@ something like::
         }
     }
 
+...note::
+    A function cannot accept a multi-dimensional array as an input parameter. This functionality is possible if you enable the experimental ``ABIEncoderV2`` feature by adding ``pragma experimental ABIEncoderV2;`` to your smart contract.
```

Comment:
Result of discussion: It might be better to create a section below the "Types" section that covers
 - which types are allowed as function parameters depending on the kind of the function 
 - how references and copies behave

Reply:
👍 See other PR comments, will create a new PR and consolidate everything.